src/server.js

```diff
@@ -59,6 +59,15 @@ export class Server {
      * @returns {Promise} Promise that resolves or rejects with response from horizon.
      */
     submitTransaction(transaction) {
+        axiosRetry(axios, this.retry);
```

Comment:
same comment as in call builder (init once at ctor, make axios instance per `Server` instance